ratelimiter/__init__.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
```

Comment:
Note that shebangs are not usefull for modules (in comparison to scripts).

Reply:
I missed that, will remove in a small commit.